node/src/main/scala/coop/rchain/node/Main.scala

```diff
@@ -14,6 +14,7 @@ import coop.rchain.metrics.Metrics
 import coop.rchain.node.configuration._
 import coop.rchain.node.diagnostics.Trace
```

Comment:
this import is not needed. this pull request does not necessarily need to fix this as its out of scope for this work.